traffic_portal/app/src/common/modules/form/topology/form.topology.tpl.html

```diff
@@ -87,7 +87,8 @@
             </div>
             <div class="modal-footer">
                 <button type="button" class="btn btn-danger" ng-show="!settings.isNew" ng-click="confirmDelete(topology)">Delete</button>
-                <button type="button" class="btn btn-success" ng-disabled="topologyForm.$pristine || topologyForm.$invalid" ng-click="save(topology.name, topology.description, topologyTree)">{{settings.saveLabel}}</button>
+                <button ng-if="settings.isNew" type="button" class="btn btn-success" ng-disabled="topologyForm.$pristine || topologyForm.$invalid" ng-click="save(topology.name, topology.description, topologyTree)">{{settings.saveLabel}}</button>
```

Comment:
Shouldn't this call to `save` have all 4 params?

Reply:
nope, the first save is for create button and second save is for update button.

Reply:
Ah, I see, they use the same function name -- that's confusing. Thank you for clarifying.

Reply:
Anytime.